sqld/src/replication/primary/logger.rs

```diff
@@ -79,10 +77,8 @@ unsafe impl WalHook for ReplicationLoggerHook {
     ) -> c_int {
         assert_eq!(page_size, 4096);
         let wal_ptr = wal as *mut _;
-        #[cfg(feature = "bottomless")]
         let last_valid_frame = wal.hdr.mxFrame;
-        #[cfg(feature = "bottomless")]
-        let _frame_checksum = wal.hdr.aFrameCksum;
+        //let _frame_checksum = wal.hdr.aFrameCksum;
```

Comment:
Why commented out and not removed? If it makes sense, please also add a comment that explains the decision, or just remove this line